core/src/main/scala/org/http4s/blaze/util/Actor.scala

```diff
@@ -62,7 +62,7 @@ private[blaze] abstract class Actor[M](
   }
 
   private[this] class Node(val m: M) extends AtomicReference[Node]
-  private[this] class RecycleableRunnable(@volatile var start: Node) extends Runnable {
+  private[this] class RecyclableRunnable(@volatile var start: Node) extends Runnable {
```

Comment:
Not sure about this change

Reply:
If it would help smooth the process, I'm happy to drop now and someone can ask questions later.